guides/common/modules/proc_upgrading-a-connected-project-server-procedure.adoc

```diff
@@ -0,0 +1,111 @@
+. Create a backup of your {ProjectServer}.
+* If your {ProjectServer} runs on a virtual machine:
+.. Stop all {Project} services:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {foreman-maintain} service stop
+----
+.. Take a snapshot.
+.. Start all {Project} services:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {foreman-maintain} service start
+----
+* If your {ProjectServer} runs on a physical machine, create a backup as described in {AdministeringDocURL}[Backing up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+ifdef::satellite[]
```

Comment:
there is no orcharino section -- there wasn't one before, so it's not like this is a regression.

@maximiliankolb I guess you're not using this guide at all and we can ignore that?